examples/thread_test/src/main.rs

```diff
@@ -0,0 +1,225 @@
+//! Inspired by the Rust standard TLS tests:
+//! <https://github.com/rust-lang/rust/tree/master/library/std/tests/thread_local>
+
+#![feature(thread_local)]
```

Comment:
Can we remove this feature? The remaining `#[thread_local]`s would have to be converted to `thread_local!`s.